tools/clang/tools/dxc/dxc.cpp

```diff
@@ -1014,11 +1014,14 @@ void DxcContext::GetCompilerVersionInfo(llvm::raw_string_ostream &OS) {
   if (m_dxcSupport.IsEnabled()) {
     UINT32 compilerMajor = 1;
     UINT32 compilerMinor = 0;
+    UINT32 commitCount = 0;
+    const char *commitHash = "<unknown-git-hash>";
     CComPtr<IDxcVersionInfo> VerInfo;
     const char *compilerName =
         m_Opts.ExternalFn.empty() ? "dxcompiler.dll" : m_Opts.ExternalFn.data();
     if (SUCCEEDED(CreateInstance(CLSID_DxcCompiler, &VerInfo))) {
       VerInfo->GetVersion(&compilerMajor, &compilerMinor);
+      VerInfo->GetCommitInfo(&commitCount, &commitHash);
```

Comment:
This is where you'd QI for the new interface to gather the rest of the fields. Also, should probably release the caller's memory once we're done with it.

Reply:
Done